### PR TITLE
Remove old extensions samples page and redirect to new one

### PR DESCRIPTION
**Pull request by AmySteam**
Remove old extensions samples page and redirect to new one

### CODE REVIEW

**Review by patrickkettner — APPROVED** [review]
LGTM

**Comment by AmySteam** [comment]
@jpmedley this is a small PR, can you take a quick look 🙏

**Review by jpmedley — CHANGES_REQUESTED** [review]
This needs to include an update of the TOC. Can you please do that.

**Comment by AmySteam** [comment]
@jpmedley PTAL

File: site/_data/docs/extensions/toc.yml [hunk 1]
```diff
@@ -44,7 +44,6 @@
     - url: /docs/extensions/mv3/service_workers
     - url: /docs/extensions/mv3/user_interface
     - url: /docs/extensions/mv3/tut_debugging
-    - url: /docs/extensions/mv3/samples
 - title: i18n.docs.extensions.service_workers
```
Comment (jpmedley) [inline site/_data/docs/extensions/toc.yml:47]:
```suggestion
    - url: /docs/extensions/samples
- title: i18n.docs.extensions.service_workers
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: site/_data/docs/extensions/toc.yml (hunk 1 above)
Comment (AmySteam) [reply site/_data/docs/extensions/toc.yml:47]: Hmm... I'm confused... Do we really need two places where we link to the samples page? 🤔


Screenshot 2023-09-26 at 12 39 33

Comment (jpmedley) [reply site/_data/docs/extensions/toc.yml:47]: My change isn't doing that. That's an existing problem we need to fix. It was probably done Oliver's new page was added. My requested change updates the left-had navigation in the content proper.

Comment (jpmedley) [reply site/_data/docs/extensions/toc.yml:47]: I'll take care of that problem.

Comment (jpmedley) [reply site/_data/docs/extensions/toc.yml:47]: I just figured out my mistake.

**Review by jpmedley — APPROVED** [review]

**Pull request merged**